From 80a5c1f9a015f1a41a76fb6c7ef56837ad192264 Mon Sep 17 00:00:00 2001
From: Sergio Lopez <slp@redhat.com>
Date: Wed, 4 Sep 2024 13:05:58 +0200
Subject: [PATCH 1/3] Revert "sommelier: bump up CROSS_DOMAIN_MAX_IDENTIFIERS"

This reverts commit 6497ca3c47ec85ceee6140d1c90ea20d415f0a82.

This change needs to be coordinated with rutabaga and libkrun. Let's
revert it until then, since it shouldn't have an impact on our current
use cases.

Signed-off-by: Sergio Lopez <slp@redhat.com>
---
 .../sommelier/virtualization/virtgpu_cross_domain_protocol.h  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/virtualization/virtgpu_cross_domain_protocol.h b/virtualization/virtgpu_cross_domain_protocol.h
index 55dff23c9c..067b79593f 100644
--- a/virtualization/virtgpu_cross_domain_protocol.h
+++ b/virtualization/virtgpu_cross_domain_protocol.h
@@ -20,8 +20,8 @@
 #define CROSS_DOMAIN_CHANNEL_TYPE_WAYLAND 0x0001
 #define CROSS_DOMAIN_CHANNEL_TYPE_CAMERA 0x0002
 
-// The maximum number of identifiers.
-#define CROSS_DOMAIN_MAX_IDENTIFIERS 28
+// The maximum number of identifiers (value based on wp_linux_dmabuf)
+#define CROSS_DOMAIN_MAX_IDENTIFIERS 4
 
 // virtgpu memory resource ID.  Also works with non-blob memory resources,
 // despite the name.
-- 
2.45.1

